src/routes.js

```diff
@@ -44,4 +45,16 @@ router.post('/coffeegrams', async ctx => {
   ctx.redirect(`/users/${ctx.state.currentUser.username}`);
 });
 
+router.post('/users/:username/bio', async ctx => {
```

Comment:
This isn't RESTful convention ;) the route would simply be `POST /users/:username` ;)